ffmpeg/videoprofile.go

```diff
@@ -16,6 +16,16 @@ const (
 	FormatMP4
 )
 
+type Profile int
+
+const (
+	ProfileNone Profile = iota
+	H264Baseline
```

Comment:
Should we use a naming scheme along the lines of `ProfileH264Baseline` ? Matches the `ProfileNone`, and follows the `Format*` convention around enum-like values.